app/views/quotes/_quote.html.erb

```diff
@@ -31,6 +31,9 @@
       <div class="flex flex-col sm:flex-row text-gray-500">
         <div class="flex-grow"><%= quote.name %></div>
         <p><%= time_ago_in_words(quote.created_at) %> geleden.</p>
+        <%= if quote.reporter %>
+          <%= heroicon('magnifying-glass', class_name: "h-4 w-4", title: "Gequote door #{quote.reporter.name}")%>
+        <% end %>
```

Comment:
```suggestion
  <%= heroicon('magnifying-glass', class_name: "h-4 w-4", title: "Gequote door #{quote.reporter_name.downcase}") %>
```

Voor consistentie zou ik altijd de tekst laten zien en dan de name een default value geven, dit zou kunnen met een functie `reporter_name` op Quote: 

```
def reporter_name
  reporter&.name || "Dinges" 
end
```

Of direct in de view zelf zonder functie in Quote: 

```
  <%= heroicon('magnifying-glass', class_name: "h-4 w-4", title: "Gequote door #{quote.reporter&.name || "``iemand``"}") %>
```

In beide gevallen gebruikt dit de "safety operator" &, die nil teruggeeft ipv de call probeert wanneer het object waarop gecalled word nil is.

Reply:
We hebben al de logic voor de anonymous ding wel of niet laten zien en hetzelfde geldt voor de reporter die dan wel of niet bestaat. Dan hoef je niet een extra "Dinges" te gaan bedenken maar laat je dan gewoon de knop niet zien.